runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerPlugin.java

```diff
@@ -250,10 +251,14 @@ public void updateConfig()
 		}
 
 		final Gson gson = new Gson();
+
 		final List<ScreenMarker> screenMarkerData = gson.fromJson(json, new TypeToken<ArrayList<ScreenMarker>>()
 		{
+			// Intentionally left empty
 		}.getType());
 
+		screenMarkerData.removeAll(Collections.singleton(null));
```

Comment:
you can simply filter the null values below instead of using removeAll. Also the extra comment is unnecessary